front/src/components/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+export function Login() {
+    const [userName, setId] = useState('');
+    const [password, setPassword] = useState('');
+    const [errorMessage, setErrorMessage] = useState('');
+    const navigate = useNavigate();
+    let { token } = require('../api.js');
+
+    function Login() {
```

Comment:
```suggestion
    function login() {
```
JSでは関数名はキャメルケースorスネークケースで書くことが多い